modin/engines/base/io/text/fwf_reader.py

```diff
@@ -116,24 +116,24 @@ def read(cls, filepath_or_buffer, **kwargs):
                     skiprows += header + 1
                 elif hasattr(header, "__iter__") and not isinstance(header, str):
                     skiprows += max(header) + 1
-                for _ in range(skiprows):
-                    f.readline()
+                cls.offset(
+                    f,
+                    nrows=skiprows,
```

Comment:
Why not `skiprows=skiprows`?

Reply:
answered [here](https://github.com/modin-project/modin/pull/1894#discussion_r481177405)